web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -175,6 +175,8 @@ public void init(FilterConfig config) {
             if (StringUtils.isNotEmpty(config.getFilterName())) {
                 this.filterName = config.getFilterName();
             }
+            long end = System.currentTimeMillis();
+            InternalLogger.INSTANCE.trace("Initialized Application Insights Filter in " + (end - start) + " ms");
```

Comment:
Should use format string.

Reply:
String concatenation is much faster over String.format. Take a look at the stack over flow article here specifically the second answer and comments on it.

https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java

Reply:
It's generally practiced to use format strings for log messages so the JVM doesn't perform concatenation when the logger is disabled.

Reply:
Makes sense. Done.